backend/src/index.js

```diff
@@ -1,7 +1,7 @@
+require('dotenv').config();
```

Comment:
@LLaszlo89 remove this line and preload the dotenv config in the start script:
`"start": "nodemon --exec babel-node -r ./node_modules/dotenv/config src/index.js",`